BitWatch/oldHomeVC.swift

```diff
@@ -0,0 +1,347 @@
+//
```

Comment:
Was just using this as a reference file. It should/will be deleted. I just wanted to read what you had.